application/controllers/admin/themes.php

```diff
@@ -657,7 +657,8 @@ public function templatesavechanges()
 
                         // If the file is an asset file, we refresh asset number
                         if (in_array($relativePathEditfile, $cssfiles) || in_array($relativePathEditfile, $jsfiles)){
-                            SettingGlobal::increaseCustomAssetsversionnumber();
+                            //SettingGlobal::increaseCustomAssetsversionnumber();
+                            Template::model()->findByPk($sTemplateName)->resetAsset();
```

Comment:
For now, assets are global, not Theme related. So the function to reset assets belongs to GlobalSetting Model, not Template Model. Also, it doesn't depend to any specific item (any instantiated object), so it should be static.